m68k: Export needed symbol for ZERO_PAGE usage in modules.

From: "Aneesh Kumar K.V" <aneesh.kumar@linux.vnet.ibm.com>

ext4 use ZERO_PAGE(0) to zero out blocks. We need to export
different symbols in different arch for the usage of ZERO_PAGE
in modules.

Signed-off-by: Aneesh Kumar K.V <aneesh.kumar@linux.vnet.ibm.com>
Signed-off-by: "Theodore Ts'o" <tytso@mit.edu>
---
 arch/m68k/mm/init.c |    1 +
 1 file changed, 1 insertion(+)

Index: linux-2.6.25-rc6/arch/m68k/mm/init.c
===================================================================
--- linux-2.6.25-rc6.orig/arch/m68k/mm/init.c	2008-03-16 16:32:14.000000000 -0700
+++ linux-2.6.25-rc6/arch/m68k/mm/init.c	2008-03-25 14:47:48.000000000 -0700
@@ -69,6 +69,7 @@ void __init m68k_setup_node(int node)
  */
 
 void *empty_zero_page;
+EXPORT_SYMBOL(empty_zero_page);
 
 void show_mem(void)
 {
